app/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE resources [
+	<!ENTITY appname "Slideshow">
+	<!ENTITY author "Michael Standen">
+	]>
+
+<resources>
+	<string name="app_name">&appname;</string>
+	<string name="action_settings">Paramètres</string>
+	<string name="action_credits">A Propos</string>
+
+	<!-- List -->
+	<string name="inaccessible">(inaccessible)</string>
+	<string name="go_home_folder">Bloqué? Retour à l'Accueil</string>
+
+	<!-- Slideshow -->
+	<string name="title_activity_image">ImageActivity</string>
+
+	<string name="delete_button">Supprimer</string>
+	<string name="delete_dialog_title">Supprimer l'image</string>
+	<string name="delete_dialog_message">Êtes-vous sûr?</string>
+	<string name="image_deleted">Image Supprimée</string>
+	<string name="image_not_deleted">Impossible de supprimer l'image</string>
+
+	<string name="share_button">Partager</string>
+	<string name="share_via">Partager par&#8230;</string>
+
+	<string name="image_detail_dimensions">Dimensions: %1$dx%2$d</string>
+	<string name="image_detail_size">Taille du fichier: %1$s</string>
+	<string name="image_detail_modified">Modifié le: %1$s</string>
+
+	<!-- Image descriptions -->
+	<string name="image_description">Image</string>
+	<string name="logo_description">Logo</string>
+
+	<!-- Credits -->
+	<string name="title_activity_credits">A Propos de &appname;</string>
+	<string name="credits_creator">by <a href="https://michael.standen.link">&author;</a></string>
+	<string name="credits_content1">Slideshow est une simple application visualiser des images sur votre téléphone.
+	</string>
+	<string name="credits_content2">Sans fioritures, cette application est la plus simple visionneuse
+		sur le marché.
+	</string>
+	<string name="credits_content3">Vous pouvez voir le code source, l'historiques des bugs ou demandes d'évolutionss sur
+		<a href="https://github.com/Screaminghawk/android-slideshow">Github</a>.
+	</string>
+	<string name="credits_content4">Cette application est libre et open source; license
+        <a href="https://tldrlegal.com/license/mit-license">MIT license</a>.
+	</string>
+	<string name="credits_content5">Merci à <a href="https://github.com/psyanite">Yan Tsui</a>
+		pour les tests et le design de l'icône &appname; .
```

Comment:
Can you please remove the space between `&appname;` and the `.`. When displayed on the about page this will look like a typo: `Slideshow .`